clientapp/src/components/Card.jsx

```diff
@@ -221,4 +221,12 @@ export class Card extends React.Component{
             return "Description";
         }
     }
-}
\ No newline at end of file
+}
+
+function translateTime(time) {
```

Comment:
can you move this inside the class or better yet, add it to the dateTime utilities file in case we need to use it in other places?